backend/api-service/src/backend_firestore.go

```diff
@@ -24,6 +24,12 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const alertsStr = "alerts"
+const inventoriesStr = "inventories"
+const inventoryTransactionsStr = "inventoryTransactions"
+const itemsStr = "items"
+const locationsStr = "locations"
```

Comment:
nit: could be more succinctly written as 
```go
const (
  alertsStr = "alerts"
  inventoriesStr = "inventories"
  // etc
)
```

Reply:
Done.

Reply:
As go readability recommend, it's better to avoid repeating the name of the type of a variable in its name. So it's better to omit the `Str` suffix in these names.

Reply:
I renamed to "Collection" from another comment from Spencer. There are already other variables with the name without any suffix so I can't just remove it, e.g. (locations).